src/components/Contacts/ContactDetails/ContactDetailsHeader/DeleteContactModal/ContactSource.graphql

```diff
@@ -1,6 +1,16 @@
-query ContactSource($accountListId: ID!, $contactId: ID!) {
+query Contact($accountListId: ID!, $contactId: ID!) {
```

Comment:
Why the rename? Honestly `ContactSource` seems like a more descriptive name for this operation.

Reply:
⬆️ @dr-bizz Did you see this comment?

Reply:
I changed it because we're getting more info than just the contactSource, we're now getting all the people on the contact